frontend/elements/src/core/popups/popup.ts

```diff
@@ -0,0 +1,77 @@
+import { MEDIA_UI } from '@utils/path';
+import { LitElement, html, css, customElement, property } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+
+
+export type Color = "peach" | "green" ;
+export type Size ="medium" | "large";
+
+
+@customElement('template-popups')
+export class _ extends LitElement {
+  static get styles() {
+    return [css`
+
+div{
+  position: relative;
+}
+
+
+  .medium{
+    width: 576px;
+    height: 352px;
+  }
+
+  .large{
+    width: 760px;
+    height: 462px;
+  }
+
+  .peach{
+    background-color:#fff2cb; 
+
+  }
+
+  .green{
+
+    background-color:#c4ead1; 
+
+  }
+
+  img-ui{
+    top:20px;
+     right:25px;
+      display:block;
+      position: absolute;
+
+  }
+    `]
+  }
+
+
+  @property()
+  color:string = ""; 
```

Comment:
color and size should be typed Color and Size respectively

Reply:
Fixed